front/lib/models/plan.ts

```diff
@@ -96,9 +96,10 @@ Plan.init(
       type: DataTypes.INTEGER,
       allowNull: false,
     },
-    maxSizeStaticDataSources: {
+    maxSizeMbStaticDataSources: {
```

Comment:
will that just work? this need a migration no?

Reply:
some would say 🤠 , some would say efficient 😄

Reply:
SGTM :+1:

Reply:
Actually not cool as I would have to run the migration at the exact moment we have the new code on prod. What I need to do: 
- One migration that creates the new column without removing the old one 
- Then I modify the code to use the new column. 
- Then I can remove the old column

Reply:
Yeah that's going to be a pain but worth. Happy to take it as eng runner

Reply:
friendly bump

Reply:
PR 1: https://github.com/dust-tt/dust/pull/2289

Reply:
PR 2: https://github.com/dust-tt/dust/pull/2290